### PR TITLE
Implementing Street Gang Recruiter

### PR DESCRIPTION
**Pull request by supercodepoet**
image

### CODE REVIEW

**Review by adbronico — DISMISSED** [review]
LGTM. Veld will review/approve and merge

**Comment by jgordijo** [comment]
LGTM. Adding the card image on the description helps giving more context while reviewing :)

**Review by jgordijo — DISMISSED** [review]

**Comment by supercodepoet** [comment]
> LGTM. Adding the card image on the description helps giving more context while reviewing :)

Sounds good, will do.

File: test/server/cards/02_SHD/units/StreetGangRecruiter.spec.ts [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+                contextRef.setupTest({
+                    phase: 'action',
+                    player1: {
+                        hand: ['street-gang-recruiter'],
+                        groundArena: ['greedo#slow-on-the-draw'],
+                        discard: ['ruthless-raider', 'ma-klounkee', 'bossk#deadly-stalker', 'hotshot-dl44-blaster']
+                    },
+                    player2: {
+                        groundArena: ['luke-skywalker#jedi-knight'],
+                        discard: ['lukes-lightsaber']
```
[the remark this answers is not in the document]
Comment (supercodepoet) [reply test/server/cards/02_SHD/units/StreetGangRecruiter.spec.ts]: That makes sense. Updated the PR with this fix.

**Review by AMMayberry1 — APPROVED** [review]

**Pull request merged**